wandb/sdk/launch/_project_spec.py

```diff
@@ -242,19 +245,21 @@ def get_single_entry_point(self) -> Optional["EntryPoint"]:
         """Returns the first entrypoint for the project, or None if no entry point was provided because a docker image was provided."""
         # assuming project only has 1 entry point, pull that out
         # tmp fn until we figure out if we want to support multiple entry points or not
-        if not self._entry_points:
+        if not self._entry_point:
             if not self.docker_image:
                 raise LaunchError(
                     "Project must have at least one entry point unless docker image is specified."
                 )
             return None
-        return list(self._entry_points.values())[0]
+        return self._entry_point
 
-    def add_entry_point(self, command: List[str]) -> "EntryPoint":
+    def set_entry_point(self, command: List[str]) -> "EntryPoint":
         """Add an entry point to the project."""
-        entry_point = command[-1]
-        new_entrypoint = EntryPoint(name=entry_point, command=command)
-        self._entry_points[entry_point] = new_entrypoint
+        assert (
```

Comment:
just curious, why `assert` over error? we are assuming this will catch developer misuse?

Reply:
yeah, the main purpose is to catch developer misuse